download_render_tools_from_golem_repository.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+
+
+display_usage(){
+    printf "Usage:\n"
+    printf " download_render_tools_from_golem_repository
+           [-r || --repository_tag=<name of tag or branch from where files should be downloaded (default: develop)>]
+           [-p || --full_path_to_store_files=<full path where downloaded files should be stored (default: /home/$USER/golem_render_tools>])\n"
+}
+
+
+ORGANIZATION_NAME="golemfactory"
+REPOSITORY_NAME="golem"
+REPOSITORY_TAG="develop"
+PATH_TO_STORE_FILES="/home/$USER/golem_render_tools"
```

Comment:
The usual convention in Bash is to user lowercase names for your own variables local to the script. Only the system and configuration variables defined in the environment (like `PATH` or `PWD`) should be capitalized.

Reply:
Changed to local variables to lowercase names